job_definitions/index_services.yml

```diff
@@ -9,8 +9,8 @@
     parameters:
       - string:
           name: INDEX
-          default: g-cloud
-          description: "Index name to use (eg 'g-cloud-2017-05-22')"
+          default: {{ default_index_name }}
```

Comment:
Because otherwise as far as I understand this job will just update unused index and the old `g-cloud` one will stay frozen. Or am I missing something?